### ISSUE TITLE
Invalid key on website

### ISSUE THREAD

**Issue by fpellet** [syncthing/syncthing#9891]
### What happened?

Hello,

The signature displayed on the website seems to have changed.
The new one doesn't match the one used to sign releases.
The old one still works though (https://web.archive.org/web/20240905195055/https://syncthing.net/release-key.txt)

```
curl -sLO https://github.com/syncthing/syncthing/releases/download/v1.28.1/sha256sum.txt.asc
curl -s https://syncthing.net/release-key.txt | gpg --import
gpg --verify sha256sum.txt.asc
```
result:
```
gpg: Signature made mar. 03 déc. 2024 12:53:08 CET
gpg:                using RSA key 37C84554E7E0A261E4F76E1ED26E6ED000654A3E
gpg: Can't check signature: No public key
```

### Syncthing version

v1.81.0

### Platform & operating system

Linux amd64

**Comment by calmh** [comment]
Yeah there's unfortunately key rotation with some bad timing going on. The asc file will be signed with the new key for next release.

**Comment by fpellet** [comment]
I see 🙂. As the old signature worked, this doesn't seem too serious. As far as I'm concerned, you can close it if you want.
One suggestion for improvement would be to name the file with the date to preserve the old ones. And to avoid breaking the links, you can redirect with the latest date.

**Comment by calmh** [comment]
This is now corrected, in that the latest release files are signed with both the old and the new key.
